collections/_blogs/2020-09-05-announcing-the-meshery-adapter-for-open-service-mesh.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title:  "Announcing the Meshery Adapter for Open Service Mesh"
+date: 2020-09-04 10:17:05 +0000
+image: /assets/images/posts/2020-09-05-announcing-the-meshery-adapter-for-open-service-mesh/meshery-adapter-for-open-service-mesh.png
+author: Lee Calcote
+categories: meshery
+permalink: /blog/:categories/:title
+---
+<h2 class="center" style="color:#666;font-size:1.25em;"> Adding another service mesh to the landscape and another adapter to Meshery</h2>
+
+[Open Service Mesh (OSM)](https://openservicemesh.io/) was released on August 5th by Microsoft. <img class="image-right" src="/assets/images/posts/2020-09-05-announcing-the-meshery-adapter-for-open-service-mesh/meshery-adapter-for-open-service-mesh.png" />Boasting some highly recommendable features, including but not limited to being open-source, lightweight, and SMI conformant, OSM is a welcome addition to the meshiness going around in the service mesh community.
+With its own control plane and a data plane built by exacting Envoy APIs, OSM is aimed to be a holistic, community-led approach to service meshes, that can be seen as a direct implementation of the SMI specification. 
```

Comment:
Not community-led. Entirely closed source until they opened it up right at the moment they can announce it.